server/app.py

```diff
@@ -0,0 +1,12 @@
+import uvicorn
+from fastapi import FastAPI
+from handlers import calendarHandler
+
+app = FastAPI()
+
+@app.get("/calendarJson")
```

Comment:
Feel like kabab-case is better for url

Reply:
Agree

Reply:
We'll leave this as a good-first-issue for someone else to fix later :)